core/interfaces/i_icon.ts

```diff
@@ -13,7 +14,7 @@ export interface IIcon {
    * E.g. 'comment', 'warning', etc. This string should also be used when
```

Comment:
I think this comment needs to be updated

Reply:
Done!